test/test_quantized_tensor.py

```diff
@@ -52,13 +52,15 @@ def test_qtensor(self):
         qr[:] = x
         self.assertEqual(qr.item(), 15)
         # we can also print a qtensor
-        self.assertEqual(str(qr),
+        self.assertEqual(' '.join(str(qr).split()),
```

Comment:
Please add a test for q_per_channel_scales() and q_per_channel_zero_points()

Reply:
Test for q_per_channel_scales() and q_per_channel_zero_points() are added in the followup diff. Please checkout 'test_per_channel_qtensor_creation' here: https://github.com/pytorch/pytorch/pull/24935/files 

quantize_per_channel and dequanize() already exists. Please check test_qtensor_per_channel_affine in this same file (test/test_quantized_tensor.py).